lua/geyser/GeyserLabel.lua

```diff
@@ -111,9 +111,15 @@ function Geyser.Label:getWindow(label)
         if v.name == label then
             return v
         end
-    end
-end
 
+        -- 030517 the below searchs down one level
```

Comment:
Might be a US date code (you know they mess up the order of months and days of the month) e.g. this person means: 2017/03/05 ... :smirk: